engine/classes/Elgg/Application.php

```diff
@@ -261,6 +263,14 @@ public function bootCore() {
 		// in case not loaded already
 		$this->loadCore();
 
+		$hooks = $this->_services->hooks;
```

Comment:
I accidentally changed the boot sequence :) Needs to be fixed whether or not this PR makes it in.